cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,53 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
+import SettingsPageObject from '../support/pages/settings.page-object';
 
-  });
+describe('Settings page', () => {
+  const settingsPage = new SettingsPageObject();
+  let user;
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.login(user.email, user.username, user.password);
+    });
+    settingsPage.visit();
   });
 
   it('should provide an ability to update username', () => {
-
+    settingsPage.usernameField.type('new');
```

Comment:
```suggestion
    settingsPage.fillTheUsernameField('new');
```
update all methods using example above